.github/workflows/os-zoo.yml

```diff
@@ -177,3 +173,19 @@ jobs:
       run: ./util/opensslwrap.sh version -c
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
+  
+  BSD-x86_64:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: cross-platform-actions/action@v0.26.0
+      with:
+        operating_system: freebsd
+        image_url: https://github.com/quarckster/freebsd-builder/releases/download/25-01-02-2/freebsd-13.4-x86-64.qcow2
```

Comment:
I created a custom image with gcc and perl5 installed and published on my personal GitHub account. I will move this repository under `opnessl` org.